swebench/harness/test_spec/test_spec.py

```diff
@@ -99,7 +111,7 @@ def get_instance_container_name(self, run_id=None):
 
     @property
     def base_dockerfile(self):
-        return get_dockerfile_base(self.platform, self.arch, self.language)
+        return get_dockerfile_base(self.platform, self.arch, self.language, **self.base_docker_specs)
```

Comment:
Allow the spec to define the php version (in future golang version, rust version, etc)